### PR TITLE
Rc validation workflow

### PR DESCRIPTION
**Pull request by elink21**

### CODE REVIEW

File: .github/workflows/run_rc_validation.yml [hunk 1]
```diff
@@ -81,8 +81,14 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: ${{ env.RC_TAG }}
-      - name: Setup GitHub CLI
-        uses: ./.github/actions/setup-gh-cli-linux
+      - id: setup-gh-cli-linux
```
Comment (dannymartinm) [inline .github/workflows/run_rc_validation.yml:84]: could you revert it to use the `./.github/actions/setup-gh-cli-linux` action please?

File: .github/workflows/run_rc_validation.yml [hunk 2]
```diff
@@ -93,11 +99,12 @@ jobs:
           touch empty_file.txt
           git add empty_file.txt
           git commit -m "Add empty file in order to create PR" --quiet
-          git push origin ${{env.WORKING_BRANCH}} --quiet
+          git push --force origin ${{env.WORKING_BRANCH}} -f --quiet
```
Comment (dannymartinm) [inline .github/workflows/run_rc_validation.yml:102]: Do we need the `force` flag? If so, could you leave only one please.

**Review by dannymartinm — CHANGES_REQUESTED** [review]

File: .github/workflows/run_rc_validation.yml (hunk 2 above)
Comment (elink21) [reply .github/workflows/run_rc_validation.yml]: In my test there were conflicts while trying to push to the destiny branch, but it was probably my branch state, I'm removing it.

File: .github/workflows/run_rc_validation.yml (hunk 1 above)
Comment (elink21) [reply .github/workflows/run_rc_validation.yml:84]: Sure, the change was for testing purposes but now I will revert it.

**Review by dannymartinm — APPROVED** [review]
LGTM! Thanks

**Pull request merged**